scripts/i18n/translate-ui.js

```diff
@@ -37,6 +37,9 @@ const progressBars = new cliProgress.MultiBar(
 	cliProgress.Presets.shades_grey
 )
 
+const consecutiveEmojiRegexp =
```

Comment:
@emilerolley quel est l'utilité de cette regex si on passe aux emojis natifs ?

Reply:
It's because combined emojis like `😶‍🌫️` are composed of `‍‍😶<200d>🌫️` and the special char `<200d>` is ignored when translated so it's need to be added at the end.